src/tactic/lift.lean

```diff
@@ -214,3 +197,9 @@ add_tactic_doc
 
 end interactive
 end tactic
+
+example (n : ℤ) (hn : 0 ≤ n) : true :=
+begin
+  lift n to ℕ, by assumption,
+  trivial
+end
```

Comment:
Should this example go in a test file?

Reply:
I think it should just be removed :rofl: